### DISCUSSION TITLE
Change default schema in PSQL

### DISCUSSION THREAD

**Discussion by swyftify** [jobrunr/jobrunr#713] [Q&A] [answered]
Currently when we deploy canaries to test, they use different schemas to the production schema. In our cases canaries run alongside production, get access to production data, but cannot write back to production. I found no way to specify the schema for the jobrunr tables, and by default they are created in public schema. Is there a way to specify which schema jobrunr tables should be created in?

Ps.
I'm using PSQL

**Comment by swyftify** [comment]
Should also mention that i am using quarkus extension

**Comment by rdehuyss** [comment, marked as answer]
Answer is mentioned in the [JobRunr documentation](https://www.jobrunr.io/en/documentation/configuration/quarkus/).

**Comment by swyftify** [comment]
Thanks, found it after a while. The parameter name isnt very clear.
